index.js

```diff
@@ -18,6 +18,12 @@ module.exports = function(env, callback) {
       nenv.addFilter(name, filter);
     });
   }
+  if(env.config.nunjucks && env.config.nunjucks.filtermodules) {
+    env.config.nunjucks.filtermodules.map( function (name) {
+      var filter = require(name);
+      filter.install(nenv);
```

Comment:
Ah interesting! I kinda see why some extensions export an `install` function, because they could export plain functions or constructors. It seems like a pretty handy API, so I'm kinda tempted to roll with it, but maybe add a note about how only functions that export an `install` function will be added?

Might also be good to post to the nunjucks mailing list or filing an issue asking if this could pose any API compatibility issues, and maybe even document it officially